<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AddressBook</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblSearch">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textSearch">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1088880</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblGroups">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Groups:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="boxGroup">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>25</height>
           </size>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>none</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777209</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionEnd"/>
   </widget>
   <widget class="QMenu" name="menuGroups">
    <property name="title">
     <string>E&amp;xtras</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglisch"/>
     <addaction name="actionPolisch"/>
     <addaction name="actionDeutsch"/>
    </widget>
    <addaction name="actionBirthdaysTable"/>
    <addaction name="separator"/>
    <addaction name="actionGroupsEdit"/>
    <addaction name="separator"/>
    <addaction name="actionImportContacts"/>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGroups"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnd">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionImportContacts">
   <property name="text">
    <string>&amp;Import contacts</string>
   </property>
  </action>
  <action name="actionGroupsEdit">
   <property name="text">
    <string>&amp;Groups edit</string>
   </property>
  </action>
  <action name="actionBirthdaysTable">
   <property name="text">
    <string>&amp;Birthdays Table</string>
   </property>
  </action>
  <action name="actionEnglisch">
   <property name="text">
    <string>Englisch</string>
   </property>
  </action>
  <action name="actionPolisch">
   <property name="text">
    <string>Polisch</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
